toronto/src/pages/Post.jsx

```diff
@@ -18,16 +16,17 @@ const Post = () => {
   });
   const [loading, setLoading] = useState(false);
   const navigate = useNavigate();
+  const token = getToken();
 
   const { title, content, agree, disagree, image } = postData;
 
   const handleClick = (e) => {
     const formData = new FormData();
     const titleData = {
-      title,
-      content,
-      agree,
-      disagree,
+      postTitle: title,
+      postContent: content,
+      agreeContent: agree,
+      disagreeContent: disagree,
     };
```

Comment:
위에서 `postData`를 수정했다면, `titleData`를 아래처럼 수정할 수 있을 것 같습니다!

```suggestion
const handleClick = (e) => {
    const formData = new FormData();
    const titleData = {
      postTitle
      postContent,
      agreeContent,
      disagreeContent,
    };
```